src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -106,8 +106,13 @@
   <Import Project="IL.targets"
            Condition="'$(MSBuildProjectExtension)' == '.ilproj' AND '$(SkipImportILTargets)'!='true'" />
 
-  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets"
-          Condition="'$(TargetFrameworkIdentifier)' == '.NETPortable' and '$(MSBuildProjectExtension)' == '.csproj'" />
+   <PropertyGroup Condition="'$(RunningOnCore)' == 'true'">
+     <CSharpCoreTargetsPath>$(MSBuildExtensionsPath32)\Roslyn\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
+     <VisualBasicCoreTargetsPath>$(MSBuildExtensionsPath32)\Roslyn\Microsoft.VisualBasic.Core.targets</VisualBasicCoreTargetsPath>
+   </PropertyGroup>
+ 
+ <!--   <Import Project="$(MSBuildThisFileDirectory)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets"
```

Comment:
Just remove this don't just comment it out. By the way if we aren't using the portable targets any more we should probably clean up some of the other related stuff about .NET Portable above.

Reply:
I see below that we are still using the .NETPortable stuff so we can clean that up as part of the move to the SDK projects.

Reply:
Right we are still using that, but I'll remove the commented out part.